psi4/src/psi4/dfocc/ccd_F_intr.cc

```diff
@@ -36,22 +36,130 @@ namespace psi {
 namespace dfoccwave {
 
 void DFOCC::ccd_F_intr() {
-    // defs
-    SharedTensor2d K, T, U, Tau;
-
-    // read
-    Tau = SharedTensor2d(new Tensor2d("T2 (Q|IA)", nQ, naoccA, navirA));
-    Tau->read(psio_, PSIF_DFOCC_AMPS);
-
-    // OO block
-    // F_mi +=  \sum_{Q,e} Tau"_ie^Q b_me^Q
-    FijA->zero();
-    FijA->contract332(false, true, navirA, bQiaA, Tau, 1.0, 1.0);
-
-    // VV block
-    // F_ae -=  \sum_{Q,m} Tau'_ma^Q b_me^Q
-    FabA->contract(true, false, navirA, navirA, nQ * naoccA, Tau, bQiaA, -1.0, 0.0);
-    Tau.reset();
+
+    // RHF
+    if (reference_ == "RESTRICTED") {
+        // defs
+        SharedTensor2d K, T, U, Tau;
+
+        // OO block
+        // F_mi = (1-\delta_{mi}) f_mi
```

Comment:
Why is the definition of the `F` intermediate changing? What other parts of the code have to change, even in the restricted case, in order for the restricted code to give the same results as before?

Reply:
I bet it's preparation for frozen virtual and fno. @bozkaya ?